pages/tag_selector.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Tag Selector
+authors:
+  - Mitchell Krome
+keywords: Tagging
+edited: 4 January 2017
+---
+
+The tag selector is opened from the view menu.
+It allows the selection of objects based on their tags.
+
+![ ](images/tag_selector.png)
+
+# Editor Layout
+
+Each row in the editor represents a new operation in the query.
```

Comment:
How about "represents a single condition in the query and its relation to other conditions" ...